aten/src/ATen/native/vulkan/ops/Mm.cpp

```diff
@@ -0,0 +1,139 @@
+#include <ATen/native/vulkan/ops/Common.h>
+#include <torch/library.h>
+
+namespace at {
+namespace native {
+namespace vulkan {
+namespace ops {
+namespace {
+
+Tensor addmm(
+    const Tensor& self_arg,
+    const Tensor& mat1_arg,
+    const Tensor& mat2_arg,
+    const Scalar beta,
+    const Scalar alpha) {
+  api::Context* const context = api::context();
+  const Tensor self = self_arg.is_vulkan() ? self_arg : self_arg.vulkan();
+  const vTensor& v_self = convert(self);
+
+  const Tensor mat1 = mat1_arg.is_vulkan() ? mat1_arg : mat1_arg.vulkan();
+  const vTensor& v_mat1 = convert(mat1);
+
+  const Tensor mat2 = mat2_arg.is_vulkan() ? mat2_arg : mat2_arg.vulkan();
+  const vTensor& v_mat2 = convert(mat2);
+
+  vTensor v_output{
+    context,
+    self.sizes(),
```

Comment:
I changed this to
`{mat1.sizes()[0], mat2.sizes()[1]},`
and the addmm test is now passing on my laptop